pennylane/fermi/fermionic.py

```diff
@@ -109,8 +114,66 @@ def __repr__(self):
         r"""Terminal representation of a FermiWord"""
         return str(self)
 
+    def __add__(self, other):
+        """Add a FermiSentence, FermiWord or constant to a FermiWord. Converts both
+        elements to be added into FermiSentences, and uses the FermiSentence __add__
+        method"""
+
+        self_fs = FermiSentence({self: 1.0})
+
+        if isinstance(other, FermiSentence):
+            return self_fs + other
+
+        if isinstance(other, FermiWord):
+            other_fs = FermiSentence({other: 1.0})
+            return self_fs + other_fs
+
+        if isinstance(other, (Number, ndarray)):
```

Comment:
Does it cover PL `numpy` as well? e.g., the following will work?

```
from pennylane import numpy as np
np.array([0.2]) + FermiWord({})
```

What happens if someone tries:

`np.array([0.2, 0.5]) + FermiWord({})`

I think that should not be allowed. The length of the ndarray should be one.

Reply:
updated now to raise an error for arrays with len>1